test/lib/tasks/update_subscription_facet_backend_data_test.rb

```diff
@@ -16,6 +16,8 @@ def test_run
       Katello::Host::SubscriptionFacet.where("id != #{@host.subscription_facet.id}").destroy_all
       Katello::Host::SubscriptionFacet.any_instance.stubs(:host).returns(@host)
       Katello::Candlepin::Consumer.any_instance.stubs(:consumer_attributes).returns(:facts => {:foo => :bar})
+      Katello::Resources::Candlepin::Consumer.stubs(:virtual_guests).returns([])
+      Katello::Resources::Candlepin::Consumer.stubs(:virtual_host).returns(nil)
```

Comment:
This is the fix for the test failure. Instead of the `Katello::Resources::Candlepin::Consumer` erroring with VCR and rescuing, we're just not going to hit the API at all.

Reply:
I did need to record a cassette for `test/services/katello/registration_manager_test.rb`.